### PR TITLE
adding highlighting tabs for used, unused defs and funs

### PR DESCRIPTION
**Pull request by sb-prateekkeerthi**
## MADiE PR

Jira Ticket: [MAT-6087](https://jira.cms.gov/browse/MAT-6087)
(Optional) Related Tickets:

### Summary
adding highlighting tabs for used, unused defs and funs

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/groupCoverage/GroupCoverage.tsx [hunk 1]
```diff
@@ -85,41 +104,49 @@ const GroupCoverage = ({ groupPopulations, calculationResults }: Props) => {
...
-        return {
-          ...populationResult,
-          populationName: FHIR_POPULATION_CODES[population.populationType],
-        };
-      });
+  const getPopulationResults = (groupId: string) => {
+    if (mappedCalculationResults) {
+      const selectedGroupCalculationResults = mappedCalculationResults[groupId];
+      if (selectedGroupCalculationResults) {
+        const relevant1Populations =
```
Comment (jkotanchik-SB) [inline src/components/editTestCase/groupCoverage/GroupCoverage.tsx]: What does the `1` in `relevant1Populations` represent?

Comment (sb-prateekkeerthi) [reply src/components/editTestCase/groupCoverage/GroupCoverage.tsx]: I was testing multiple way in which I was trying to get the required output so I named the relevantPopulations, relevant1Populations and after finalizing the method I forgot to remove the number. I'll remove it

**Review by jkotanchik-SB — APPROVED** [review]
